apps/test/unit/lib/kits/maker/commandsTest.js

```diff
@@ -42,61 +43,68 @@ describe('maker/commands.js', () => {
     it('display warning when reserved pin 1 is used', () => {
       pinMode({pin: 1, mode: 'input'});
       expect(errorHandler.outputWarning).to.have.been.calledWith(
-        'pinMode() pin parameter value (1) is a reserved pinid. Please use a different pinid'
+        'pinMode() pin parameter value (1) is a reserved pinid. Please use a different pinid.'
       );
     });
-
+ 
+    it('display error when invalid pin 13 is used', () => {
+      pinMode({pin: 13, mode: 'input'});
+      expect(
```

Comment:
I think you’ll want to do something similar to the test above and verify outputError here as well, e.g. expect(errorHandler.outputError).to.have.been.calledWith(…)

Reply:
I initially had this, but there was an error due to an undefined. When the parameter is not valid, there is an early return and board.pinMode is not actually called. It it is, then user receives two different error messages.

Reply:
We resolved the error offline!